Source/WebInspectorUI/UserInterface/Models/CSSGrouping.js

```diff
@@ -80,6 +105,31 @@ WI.CSSGrouping = class CSSGrouping
         console.assert(this.isMedia);
         return "@media";
     }
+
+    // Private
+
+    _handleChangeGroupingTextResult(groupingPayload)
+    {
+        console.assert(groupingPayload.type == this._type);
+
+        console.assert(groupingPayload.ruleId);
```

Comment:
since this is coming from the protocol, it shouldn't be necessary to assert that these things exist